velox/exec/WindowFunction.h

```diff
@@ -111,10 +111,11 @@ class WindowFunction {
       const core::QueryConfig& config);
 
  protected:
-  // Set result NULL for rows with invalid frames in the input.
-  void setNullEmptyFramesResults(
+  // Set result to default value for rows with empty frames.
```

Comment:
It is not obvious what "default value" is. Are you referring to the value produced by SELECT agg(...) FROM <empty table> ?

Reply:
Thanks for the feedback, yes it is referring to the aggregate value returned when there are no input rows. Updated the comment.